source/platforms/github/_tests/_github_api.test.ts

```diff
@@ -363,9 +363,26 @@ describe("Peril", () => {
       delete process.env.PERIL_BOT_USER_ID
     })
 
-    it("Makes getUserId return undefined", async () => {
+    it("Makes getUserId return PERIL_BOT_USER_ID", async () => {
```

Comment:
I believe shouldn’t always return `PERIL_BOT_USER` here, instead repeat this test case:

1. A test case that matches the old test name where all vars are unset, `GITHUB_WORKFLOW, GHE_ACTIONS_BOT_USER_ID, PERIL_BOT_USER = undefined`
2. `PERIL_BOT_USER` if nobody is logged in (`GHE_ACTIONS_BOT_USER_ID` is unset, but `GITHUB_WORKFLOW` is set, and `PERIL_BOT_USER` is set)
3. `41898282` if `PERIL_BOT_USER` is unset, `GHE_ACTIONS_BOT_USER_ID` is unset, but `GITHUB_WORKFLOW` is set.
4.  The test case you’ve already added below`"Allows setting GHE_ACTIONS_BOT_USER_ID env variable"`

Reply:
I think the tests should now be updated to cover all scenarios.